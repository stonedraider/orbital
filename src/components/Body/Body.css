.Body {    
    flex: 1;
    /*background: rgba(1,1,1,0.15);*/
    background: black;
}

.divTop {
    padding-top: 40px;
}

.divSlider {
    margin: 0 auto;
    padding: 0 50px 0 50px;
    color: #333;
}

.slick-dots {
    bottom: -60px;
}
.slick-dots li button:before,
.slick-dots li.slick-active button:before {
    color: white !important;
}
.slick-initialized .slick-slide {
    margin: 0 10px 0 10px;
}

img {
    margin-left: auto;
    margin-right: auto;
    max-width: 100%;
}

hr {
    border-top: 1px solid rgba(255,255,255,1);
    margin: 12px 80px 12px 80px;
}

.divSliderImage {
}

.divSliderItemInfo {
    height: 240px;
    border: 1px solid rgba(0,0,0,0.15);
    padding: 0 0 25px 0;
    color: white;
}

.divDownloadButton {
    padding-top: 20px;
}

iframe {
    margin-left: auto;
    margin-right: auto;
    max-width: 100%;
    height: auto;
}

/*@media (max-width: 1980px) {
    .divSliderItemInfo {
        display: none;
    }
    .divSliderImage {
        width: 100%;
    }
    hr {
        display: none;
    }
    .divTop {
        padding-top: 10px;
    }
    iframe {
        width: auto;
        height: auto;
        min-width: 600px;
        min-height: 450px;
    }
}*/