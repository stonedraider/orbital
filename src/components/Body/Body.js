import React, { Component } from 'react';
import './Body.css';

function Body() {
    return (
      <div className="Body">
        Helsdsdfdflo!
      </div>
    );
}

export default Body;