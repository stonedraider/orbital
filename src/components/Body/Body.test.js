import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Body from './Body';

describe('Body', () => {

    it('Body renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Body />, div);
    });

    test('snapshots', () => {
        const component = renderer.create(
            <Body />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});