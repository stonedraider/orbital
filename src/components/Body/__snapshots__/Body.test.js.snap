// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body snapshots 1`] = `
<div
  className="Body"
>
  Helsdsdfdflo!
</div>
`;
